src/service.ts

```diff
@@ -27,6 +27,8 @@ export * from 'vscode-json-languageservice'
 
 interface ASLLanguageServiceParams extends LanguageServiceParams { aslOptions?: ASLOptions }
 
+export const ASLSCHEMA = aslSchema as JSONSchema
```

Comment:
I'd change it to `ASL_SCHEMA` for better clarity. Also, you can export any other functions that could be reused in other contexts.

Reply:
I have now added this change to a new commit